modules/apps/analytics/analytics-client-js/src/analytics.js

```diff
@@ -127,6 +126,16 @@ class Analytics {
 		ENV.Analytics.create = Analytics.create;
 		ENV.Analytics.dispose = Analytics.dispose;
 
+		if (
+			Liferay &&
+			Liferay.ThemeDisplay &&
+			Liferay.ThemeDisplay.getUserEmailAddress()
+		) {
```

Comment:
There is possibility to use optional chaining in this condition?

Reply:
optional chaining isn't supported in internet explorer and it won't work on certain versions of other browsers older than 2020.  

I'd be a little worried about using it since I think we'd want the client to be able to run on almost any browser.